DfE.Hexagonal.Application/UseCases/School/GetAllSchools/GetAllSchools.cs

```diff
@@ -0,0 +1,29 @@
+﻿using DfE.Hexagonal.Application.Shared.Repositories;
+using DfE.Hexagonal.Common;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DfE.Hexagonal.Application.UseCases.School.GetAllSchools
```

Comment:
Given the choice, my personal preference is for file-scoped namespaces.

Do we have a team view on this?

Reply:
fine by me